src/styles/blocks/card.css

```diff
@@ -0,0 +1,125 @@
+.card {
+  box-sizing: border-box;
+
+  max-width: 200px;
+
+  padding: 0 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
no empty lines are needed here.
More info here https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/